src/Wallabag/CoreBundle/Resources/views/themes/material/footer.html.twig

```diff
@@ -1,37 +1,4 @@
 <footer class="page-footer cyan darken-2">
-    <div class="container">
-        <div class="row">
-            <div class="col l6 s12">
-                <h5 class="white-text">{{ 'footer.wallabag.elsewhere'|trans }}</h5>
-                <p class="grey-text text-lighten-4">
-                    <a target="_blank" class="grey-text text-lighten-3" href="{{ addonsUrl.google_play }}" title="Android">
```

Comment:
Do we still need `addonsUrl` & `addonsUrl` to injected as global variable?
Baggy doesn't seems to use them.